docs/managers/version.rst

```diff
@@ -74,6 +74,22 @@ Configuration
       - string
       - "CLUSTER_SIMILAR"
       - see Glyphs `README`_ for a list of supported formatters
+    * - pr_releases
+      - boolean
+      - true
+      - automatically create releases from appropriately labeled PRs
+    * - release_label_config
```

Comment:
identation with others is not aligning

Reply:
in the docs view, it show the pr_releases and release_label_config in same block